### PR TITLE
Say ‘team member’ not ‘user’

### PR DESCRIPTION
**Pull request by quis**
This is currently inconsistent with the language we use everywhere else.

### CODE REVIEW

File: app/templates/views/edit-user-permissions.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
         {{ page_footer(
           'Save',
           delete_link=url_for('.edit_user_permissions', service_id=current_service.id, user_id=user.id, delete='yes'),
-          delete_link_text='Remove user from service'
+          delete_link_text='Remove team member from service'
```
Comment (karlchillmaid) [inline app/templates/views/edit-user-permissions.html]: Should we say 'this service'?

Comment (quis) [reply app/templates/views/edit-user-permissions.html]: Options:
***
1. 
  image
2.
  image
3.
  image
4.
  image
5.
  image

I think 2. is my preference…

Comment (karlchillmaid) [reply app/templates/views/edit-user-permissions.html]: Agreed.

Comment (quis) [inline app/templates/views/edit-user-permissions.html]:
```suggestion
          delete_link_text='Remove this team member'
```

**Pull request merged**